README.md

```diff
@@ -1,13 +1,33 @@
-# Project Name
+# Project: Guess Who? week2
 
-Replace this readme with your own information about your project.
+This week we are building a classic "Guess Who" game and digging deaper into JavaScript 
+with objects, arrays and functions. This weeks project was really difficult.
+
+Some of the learnings:
+
+- How to plan and think about game logic
+- Create and manipulate loops, objects and arrays.
+- Array methods such as `forEach()` and `filter()`
+- Use built in functions, map and math.random etc.
+- the differents between "including value" and "value"
+- More about manipulating the DOM using Javascript
+- How to structure your code in functions
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I know of the game, we have it at home and play alot, that was the easy part. I started whit the pre written code, trying to find out what was missing, what the different steps should be and goggled (W3 and MDN), with "trial and error" found a way forward to a solution. I have struggled with the filter functions and arrays. 
```

Comment:
Also a clear explanation of your process for the project, your challenges and what the next steps would be if you had more time to build on this project.